docs/running-in-ci.md

```diff
@@ -0,0 +1,239 @@
+        #env:
+        #  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+        with:
+          publishToken: ${{ secrets.SEMGREP_TOKEN }}
+          publishDeployment: ${{ secrets.SEMGREP_DEPLOYMENT_ID }}
+
+          # never fail the build due to findings on pushes, but collect findings data
+          #auditOn: push
+
+          # Generate a SARIF file for GitHub's code scanning feature. See the next step.
+          #generateSarif: "1"
+
+      # Upload SARIF file generated in previous step
+      #- name: Upload SARIF file
+      #  uses: github/codeql-action/upload-sarif@v1
+      #  with:
+      #    sarif_file: semgrep.sarif
+      #  if: always()
+```
+
+</p>
+</details>
+
+<a name="inline-pr-comments-beta"></a>
+<br />
+
+### PR Comments (beta)
+
+!!! info
+    This feature is currently only available for GitHub.
+
+To get inline PR comments on your pull requests, set the `GITHUB_TOKEN` environment variable in your workflow file to `secrets.GITHUB_TOKEN`, which is the GitHub app installation access token and takes the form of this snippet:
+
+```
+uses: returntocorp/semgrep-action@v1
+        env: # Optional environment variable for automatic PR comments (beta)
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+```
+
+See a complete example of this workflow file including this environment variable (commented out) in the [above example workflow file](#github-actions).
+
+!!! info
+    Unlike `secrets.SEMGREP_TOKEN`,
+    there’s no need to create `secrets.GITHUB_TOKEN` yourself
+    because it’s automatically set by GitHub.
+    It only needs to be passed to the action via the workflow file.
+
+Comments are left when Semgrep CI finds a result that blocks CI.
+Note that this feature is experimental; please reach out to support@r2c.dev to report any issues.
+<br /><br />
+
+## GitLab CI
+
+<p>
+
+```yaml
+include:
+  - template: "Workflows/MergeRequest-Pipelines.gitlab-ci.yml"
+
+semgrep:
+  image: returntocorp/semgrep-agent:v1
+  script:
+    - python -m semgrep_agent --publish-deployment $SEMGREP_DEPLOYMENT_ID --publish-token $SEMGREP_TOKEN
+```
+
+</p>
+</br>
+
+## Other providers
+
+Although not fully supported, these instructions are here to help you integrate with your CI provider of choice.
+
+The following commands can be run by your CI provider (or on the commandline):
+
+<p>
+
+```sh
+# Set additional environment variables
+$ SEMGREP_JOB_URL=https://example.com/me/myjob
+$ SEMGREP_REPO_URL=https://gitwebsite.com/myrepository
+$ SEMGREP_BRANCH=mybranch
+$ SEMGREP_REPO_NAME=myorg/myrepository
+
+# Run semgrep_agent
+$ python -m semgrep_agent --publish-deployment $SEMGREP_DEPLOYMENT_ID --publish-token $SEMGREP_TOKEN
+```
+
+</p>
+
+For diff-aware scans, include the flag `--baseline-ref` set to a git ref (branch name, tag, or commit hash) to use as a baseline. This will prompt Semgrep to ignore findings that were already present in the codebase, and only show findings that were introduced by modifications to the baseline.
```

Comment:
“…will prompt Semgrep to ignore findings that were…” ← should this be Semgrep CI or Semgrep Agent?